### PR TITLE
MISUV-4772 : Repayments enable logic by FS

### PR DESCRIPTION
**Pull request by AshokGitAccount**

### CODE REVIEW

**Comment by avelx** [comment]
@AshokGitAccount  - please double check story and make sure you understand it correctly:
https://jira.tools.tax.service.gov.uk/browse/MISUV-4755

**Comment by AshokGitAccount** [comment]
Changes made and added test case

File: app/controllers/CreditAndRefundController.scala [hunk 1]
```diff
@@ -202,6 +209,8 @@ class CreditAndRefundController @Inject()(val authorisedFunctions: FrontendAutho
   private def handleStatusRefundRequest(isAgent: Boolean, itvcErrorHandler: ShowInternalServerError, backUrl: String)
                                  (implicit user: MtdItUser[_], hc: HeaderCarrier, ec: ExecutionContext, messages: Messages): Future[Result] = {
     repaymentService.view(user.nino).flatMap {
+      case _ if isDisabled(CreditsRefundsRepay) =>
```
Comment (avelx) [inline app/controllers/CreditAndRefundController.scala:212]: we really should check is FS before we are calling repayment service

Comment (avelx) [inline app/controllers/CreditAndRefundController.scala:212]: we really should check is FS before we are calling repayment service

**Review by avelx — APPROVED** [review]
looks ok

**Pull request merged**